scripts/master_boostrap.sh

```diff
@@ -0,0 +1,428 @@
+#!/bin/bash -e
```

Comment:
This should probably go to `deploy/scripts` directory instead. And there's already some `setup-salt-master-pkg.sh`.